biodatasets/verspoor_2013/verspoor_2013.py

```diff
@@ -0,0 +1,252 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+TODO:
+Description
+"""
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from utils import parsing, schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_CITATION = """\
+@article{verspoor2013annotating,
+title={Annotating the biomedical literature for the human variome},
+author={Verspoor, Karin and Jimeno Yepes, Antonio and Cavedon, Lawrence and McIntosh, Tara and Herten-Crabb, Asha and Thomas, Zo{"e} and Plazzer, John-Paul},
+journal={Database},
+volume={2013},
+year={2013},
+publisher={Oxford Academic}
+}
+"""  # noqa: E501
+
+
+_DATASETNAME = "verspoor_2013"
+
+_DESCRIPTION = """\
+This dataset is designed for XXX NLP task.
+"""
+
+_HOMEPAGE = ""
+
+# The source repo (https://github.com/rockt/SETH/blob/master/LICENSE) is license
+# Apache 2.0 but does that match the dataset? Best info I have for now...
+_LICENSE = """\
```

Comment:
> he source repo (https://github.com/rockt/SETH/blob/master/LICENSE) is license
 Apache 2.0 but does that match the dataset?

Exactly: we do not know. I would put a `utils.constants.BigBioValues.NULL` here

Reply:
Got it!